appserver/monitoring-console/webapp/pom.xml

```diff
@@ -115,7 +122,11 @@
                         <sources>
                             <source>src/main/webapp/mc-main.js</source>
                             <source>src/main/webapp/mc-model.js</source>
+                            <source>src/main/webapp/mc-view-components.js</source>
```

Comment:
Static resources are recommended to place in separate folder e.g resources/font, resources/image, resources/css etc. 
Example: It can help in defining the cache filter by URL pattern of the static resources.